README.md

```diff
@@ -1,16 +1,35 @@
 # Practical Tools for Simple Design
 
+###### Officially Supported Platforms and IDEs
+
+|                                | Windows | macOS | Linux |
+|:------------------------------:|:-------:|:-----:|:-----:|
+|             CLion              |    V    |   V   |   V   |
+|        VSCode[^codeoss]        |    V    |   V   |   V   |
+| Visual Studio[^novs][^vsmacos] |    ?    |   X   |   X   |
+|             No IDE             |    V    |   V   |   V   |
+
+[^vsmacos]: [Microsoft Kills Visual Studio for Mac](https://visualstudiomagazine.com/articles/2023/08/30/vs-for-mac-retirement.aspx)
+[^codeoss]: On Linux the support of Code - OSS and VSCodium aren't guaranteed.
+[^novs]: We don't provide support for Visual Studio, but building PTSD on VS is available.
```

Comment:
I feel like we could provide partial support, but due to lack of testing there may or may not be more issues on VS

Reply:
```suggestion
[^novs]: We could provide partial support, but due to lack of testing there may or may not be more issues on VS. anyway, building PTSD on VS is available.
```